packages/types/src/transport/index.ts

```diff
@@ -1,8 +1,57 @@
-import { ChildProcess } from 'child_process';
-import { TransportMessageHandler } from './structs';
+import { EventEmitter } from 'events';
+import * as net from 'net';
+import { TransportMessageHandler} from './structs';
 
 type RemoveHandlerFunction = () => void;
 
+export interface ITransportChild extends EventEmitter {
+    send(message: any, callback?: (error: Error) => void): boolean;
+    kill(signal?: string): void;
+
+    addListener(event: string, listener: (...args: any[]) => void): this;
+    addListener(event: "close", listener: (code: number, signal: string) => void): this;
+    addListener(event: "disconnect", listener: () => void): this;
+    addListener(event: "error", listener: (err: Error) => void): this;
+    addListener(event: "exit", listener: (code: number, signal: string) => void): this;
+    addListener(event: "message", listener: (message: any, sendHandle: net.Socket | net.Server) => void): this;
```

Comment:
Please, remove unused API parts (like `sendHandle` here), it will be more clearer